packages/protocol/contracts/L1/TaikoConfig.sol

```diff
@@ -11,23 +11,23 @@ import {TaikoData} from "../L1/TaikoData.sol";
 library TaikoConfig {
     function getConfig() internal pure returns (TaikoData.Config memory) {
         return TaikoData.Config({
-            chainId: 167,
-            // Two weeks if avg block time is 10 seconds
-            maxNumProposedBlocks: 120960,
-            ringBufferSize: 120960 + 10,
+            chainId: 167005,
+            // Two weeks if avg block time is 3 seconds (is it reasonable?)
```

Comment:
I think it is reasonable. How is 403200 calculated?

Reply:
its calculated  by `20 * 60 * 24 * 14` (avg block time is 3 seconds => 20 blocks per minute).